miner/Miner.hs

```diff
@@ -153,6 +170,7 @@ pCommand = hsubparser
     (  command "cpu" (info cpuOpts (progDesc "Perform multicore CPU mining"))
     <> command "gpu" (info gpuOpts (progDesc "Perform GPU mining"))
     <> command "keys" (info keysOpts (progDesc "Generate public/private key pair"))
+    <> command "balance-check" (info balanceCheckOpts (progDesc "Get balances on all chains"))
```

Comment:
Bikeshedding suggestion: `balances` instead of `balance-check`

Reply:
Or singular `balance` even.